Setting up MongoDB connection

sarahzacky opened the pull request:
Setting up MongoDB connection

sarahzacky committed:
Setting up MongoDB connection

Setting up MongoDB connection

danielskjo commented on backend/server.js:
Does this connection work even without the line
`dotenv.config();`?

danielskjo commented on .env:
I feel like we should make at least the username and password of this MONGO_URI private. Maybe add the .env file to .gitignore?

danielskjo committed:
Sync (#8)

* Added project README

* TWEEL-94-As-a-developer-I-want-to-set-up-Shadcn-on-the-front-end-so-that-I-can-utilize-reusable-components-when-implementing-the-UI (#5)

* Shadcn Installed

Vanilla Next.js and Shad installed

* need to clean

* Delete tailwind.config.js

* Revert "Delete tailwind.config.js"

This reverts commit 3dcfa6e32613ed98296fd42cb8755cc6dd824adb.

* disabled CSS Variables (working)

* Install backend dependencies, set up express server (#6)

---------

Co-authored-by: Diego O Garcia <59929620+diegogarciacs@users.noreply.github.com>
Co-authored-by: Grant Smith <59898052+grantsmith01@users.noreply.github.com>

sarahzacky committed:
Consolidating files

Moving db.js and server.js to the "server" folder
Also removed the "backend" folder

sarahzacky committed:
Merge branch 'main' into TWEEL-13-Set-up-MongoDB

danielskjo commented on server/server.js:
Let's move this code into app.js and delete the server.js file

sarahzacky committed:
Consolidate files

Delete server.js and move contents into app.js

sarahzacky committed:
Connects MongoDB but has issue with reading .env variables

sarahzacky committed:
Move .env into server folder

Now we can access variables from .env within the server folder by using "process.env.___"

sarahzacky committed:
Updated db.js

Remove a line that was used for debugging
